code-challenges.test.js

```diff
@@ -49,4 +110,26 @@ const hand3 = [5, 5, 5, 5, 4]
 const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
+describe("fullHouse", () => {
+    it("returns true or false based on if the array is a full house", () => {
+        expect(fullHouse(hand1)).toEqual("true")
+        expect(fullHouse(hand2)).toEqual("true")
+        expect(fullHouse(hand3)).toEqual("false")
+        expect(fullHouse(hand4)).toEqual("true")
+    })
+})
+
 // b) Create the function that makes the test pass.
+
```

Comment:
Let's break this one down a little bit. I want to do this simply so I will use conditionals. The if and else ifs need to match the given arrays with conditions that meet a "full house" (given in the prompt). Match the array indexes and return a true if it is a full house. Else (or everything else) returns false because it is not a full house.